init.fish

```diff
@@ -78,6 +80,8 @@ abbr -a gstd       git stash drop
 abbr -a gstp       git stash pop
 abbr -a gsts       git stash show --text
 abbr -a gsu        git submodule update
+abbr -a gsw        git switch
+abbr -a gswc       git switch -c
```

Comment:
I'm not the maintainer of this repo but I think it would be usefull to alias the explicitely option to inform the user what he's doing => `git switch --create`

Reply:
Yes, please use long form option names where feasible.